src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -267,15 +178,65 @@ public final boolean isActive() {
 
     @Nonnull
     private List<FlowNode> loadParents(List<String> parentIds) {
-        List<FlowNode> _parents = new ArrayList<>(parentIds.size());
-        for (String parentId : parentIds) {
+        List<FlowNode> _parents;
```

Comment:
Why not move declaring `_parents` into the `else` block, since it's not referenced elsewhere?